src/functions/inputValidator/validateChangePasswordForm.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Validate user input - change password form
+ *
+ * @author Hyecheol (Jerry) Jang <hyecheo123@gmail.com>
+ */
+
+import Ajv from 'ajv';
+
+export const validateChangePasswordForm = new Ajv().compile({
+  type: 'object',
+  properties: {
+    currentPassword: {type: 'string'},
+    newPassword: {type: 'string'},
```

Comment:
add password double-check here?
Since it's a part of validation...

Reply:
This is to check whether response body "contains" newPassaord and currentPassword.